dns_managed_zone_with_wildcard_certificate/main.tf

```diff
@@ -0,0 +1,112 @@
+/**
+ * Copyright 2023 Google LLC
```

Comment:
Please use a branch other than main

Reply:
open a new pull request with wildcard-cert branch
 https://github.com/terraform-google-modules/terraform-docs-samples/pull/263
this one can be closed